spec/lib/krikri/harvester_spec.rb

```diff
@@ -1,19 +1,61 @@
 require 'spec_helper'
 
 describe Krikri::Harvester do
-  it 'is a havester' do
-    expect(subject).to be_a Krikri::Harvester
-  end
+
+  # Our subject is an instance of a dummy class that mixes in
+  # Krikri::Harvester.
+  let(:klass) { Class.new }
+  subject { klass.include(Krikri::Harvester).new }
 
   context 'with record_ids implemented' do
     before do
-      described_class.send(:define_method, :record_ids) do
-        [1, 2, 3, 4, 5]
-      end
+      expect(subject).to receive(:record_ids).and_return([1, 2, 3, 4, 5])
```

Comment:
I think that this was only to support the 'knows its record count' example.

My assumption, from the way it had been written, was that `#count` invokes `#record_ids`.  I was probably also going by the `context`, which says, "with record_ids implemented."  I didn't know about `about`, though, and I see now how it's softer than `expect`.  I could change it.  However, I wonder if doing so would let us put tests in this context that shouldn't be there.  If they never called `record_ids` then should they be in this `context` block?  Maybe `expect` would flag that.  I'm not super invested in `expect`, whatever the case.